app/build.gradle.kts

```diff
@@ -48,14 +48,14 @@ android {
             isShrinkResources = true
             proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
             signingConfig = signingConfigs[BuildType.RELEASE]
-            buildConfigField("String", "BASE_API_URL", "\"https://jsonplaceholder.typicode.com/\"")
+            buildConfigField("String", "BASE_API_URL", "\"https://api.coingecko.com/api/v3/\"")
```

Comment:
@hoangnguyen92dn No biggie: What do you think if we create a `Secret` file inside the directory `production` and `staging` and put the `BASE_API_URL` as `cont val` inside there instead of using `buildConfigField`? I feel like it's more flexible in the future but I'm good to set in `buildConfigField` too. cc @luongvo

Reply:
@Wadeewee Makes sense, but please concentrate on Compose and Integration part instead. We should not spend much time on other parts 🙏